cran-comments.md

```diff
@@ -0,0 +1,44 @@
+## Maintainer change
```

Comment:
In brms I have cran-comments.md under .gitignore. I remember Hadely Wickham recommending to not commit it but it is some time ago that I have read it. Could you check how tidyverse packages to it?

Reply:
I was also unsure about this, but all the tidyverse packages I just checked (ggplot2, dplyr, and tidyr) have a `cran-comments.md` in their repo and not in `.gitignore`. And when calling `usethis::use_cran_comments()` in a new package with a Git structure, `cran-comments.md` is not added to `.gitignore`. So I think it's ok not to `.gitignore` it.

Reply:
Ok, thank you!